shared/src/commonMain/kotlin/org/hyperskill/app/notification_onboarding/domain/analytics/NotificationsOnboardingClickedAllowNotificationsHyperskillAnalyticsEvent.kt

```diff
@@ -0,0 +1,28 @@
+package org.hyperskill.app.notification_onboarding.domain.analytics
```

Comment:
I suppose package name should be `notification_onboarding` -> `notifications_onboarding`